frontend/src/libs/hooks/use-app-stepper/use-app-stepper.hook.ts

```diff
@@ -0,0 +1,24 @@
+import { useState } from '~/libs/hooks/hooks.js';
+
+type Parameters = {
+  stepsLength: number;
+};
+
+type ReturnValue = {
+  currentStep: number;
+  isFirstStep: boolean;
+  isLastStep: boolean;
+  changeStep: (step: number) => void;
+};
+
+const useStepper = ({ stepsLength }: Parameters): ReturnValue => {
+  const [currentStep, setCurrentStep] = useState(1);
+  const isFirstStep = currentStep === 1;
```

Comment:
```suggestion
  const [currentStep, setCurrentStep] = useState(CONSTANT_NAME);
  const isFirstStep = currentStep === CONSTANT_NAME;
```
add constants for it